app/models/grda_warehouse/hud/project.rb

```diff
@@ -76,44 +76,92 @@ class Project < Base
     # Needs to come after has_many :enrollments, bc one extension uses a has_many through: :enrollments relation
     include RailsDrivers::Extensions
 
+    # A scope to return any projects that are residential (provide housing).  Generally, and
```

Comment:
Can we add a comment about why Street Outreach is included? It will always confuse me that it's considered residential in the codebase, since docs describe Street Outreach projects as specifically providing `non- facility-based care`.

Reply:
Oops, guess I was too late

Reply:
Can do!

Reply:
@gigxz does this suffice?
https://github.com/greenriver/hmis-warehouse/pull/5125/commits/d713fb2f1ab4639107dc5d337e8f79a80f325f12

Reply:
perfect thanks!